apps/charterafrica/src/payload/collections/Grants.js

```diff
@@ -0,0 +1,104 @@
+import content from "../fields/content";
+import richText from "../fields/richText";
+import slug from "../fields/slug";
+
+const Grant = {
+  slug: "grants",
+  admin: {
+    useAsTitle: "title",
+    defaultColumns: ["title", "author", "publishedOn"],
+  },
+  access: {
+    read: () => true,
+  },
+  versions: {
+    drafts: true,
+  },
+  fields: [
+    {
+      name: "title",
+      label: {
+        en: "Title",
+        fr: "Titre",
+        pt: "Título",
+      },
+      type: "text",
+      localized: true,
+      required: true,
+    },
+    slug(),
+    {
+      name: "coverImage",
+      label: {
+        en: "Cover Image",
+        pt: "Imagem de capa",
+        fr: "Image de couverture",
+      },
+      type: "upload",
+      relationTo: "media",
+      required: true,
+      filterOptions: {
+        mimeType: { contains: "image" },
+      },
+    },
+    {
+      name: "status",
+      label: {
+        en: "Status",
+        fr: "Statut",
+        pt: "Status",
+      },
+      type: "select",
+      options: [
+        {
+          label: {
+            en: "Open",
+            fr: "Ouvrir",
+            pt: "Abrir",
+          },
+          value: "open",
+        },
+        {
+          label: {
+            en: "Closed",
+            fr: "Fermé",
+            pt: "Fechado",
+          },
+          value: "closed",
+        },
+        {
+          label: {
+            en: "Upcoming",
+            pt: "Por vir",
+            fr: "A venir",
+          },
+          value: "upcoming",
+        },
+      ],
+    },
+    richText({
+      name: "excerpt",
+      label: {
+        en: "Excerpt",
+        fr: "Extrait",
+        pt: "Excerto",
+      },
+      localized: true,
+      admin: {
+        elements: ["leaves"],
+      },
+    }),
+    content(),
+    {
+      name: "deadline",
+      type: "date",
+      required: true,
```

Comment:
Just thinking out loud, if a grant is upcoming, does it need a deadline? Shouldn't deadlines only be required if status is open/closed?